### ISSUE TITLE
Kirby Triple Deluxe [000400000010C000]

### ISSUE THREAD

**Issue by togguleru** [Lime3DS/compatibility-list#8]
### Prerequisites

- [X] I have checked there is no currently open report for this game. (An updated report can be made for games with previously closed reports).
- [X] I have read through the contribution guide [here](https://github.com/Lime3DS/compatibility-list/blob/master/CONTRIBUTING.md) from start to finish.
- [X] I have played through this game from start to finish only using official releases of the Lime3DS emulator.
- [X] The OpenGL renderer was used for my entire playthrough.

### Rating

Won't Boot

### Description

Emulator crashes when trying to boot up the game.

### Lime3DS Version

2114

### System Configuration

CPU: AMD Ryzen 7 4800H
GPU/Driver: NVIDIA GeForce GTX 1650 Ti Mobile
RAM: 16GB
OS: Linux Mint 21.3

**Comment by MontyJack2020** [comment]
I disagree with this report.

I would rate this as: Okay to Great

Description
I was able to launch the game several times with no crashes - I have only played thought the first world but had no issues - some stutter while graphics shader cache was being built. I played at 2x and 3x resolution and experienced little to no stutters.

Lime3DS Version
2117.1

System Configuration
CPU: AMD Ryzen 7 PRO 3700 8 Core processor 3.60 GHz
GPU/Driver: NVIDIA GeForce GT 1030
RAM: 32GB
OS: Microsoft Windows 10 22H2 19045.2965

![IMG_20240904_085732](https://github.com/user-attachments/assets/83217f73-2be4-4789-9bd5-596139277a71)
